### DISCUSSION TITLE
how to execute commands on stream end?

### DISCUSSION THREAD

**Discussion by davidhu2000** [bluenviron/mediamtx#912] [Questions] [answered]
I see there is a section on saving fragments on disk, https://github.com/aler9/rtsp-simple-server#save-streams-to-disk 

following the instructions I end up with multiple pieces of videos that need to be stitched together and processed like uploading to s3. 

what's the recommended way to run the process function? I'm not seeing anything in the config itself for after stream ends.

**Comment by davidhu2000** [comment, marked as answer]
@aler9 can you help with this question? thank you

**Reply by aler9** [reply]
please read here: https://github.com/aler9/rtsp-simple-server/discussions/554#discussioncomment-1282935

just replace `runOnPublish` with `runOnReady`

**Reply by davidhu2000** [reply]
ah perfect. thank you very much

[a comment by an automated account is not shown]
